### DISCUSSION TITLE
A web mail

### DISCUSSION THREAD

**Discussion by IbrahimDevX** [is-a-dev/register#1152] [Ideas]
So basically you could have a domain like luke.is-a.dev and get a free luke@is-a.dev email to go with it.

**Comment by TheBotlyNoob** [comment]
Implemented in #1093, for security risks you cannot get a top level `@is-a.dev` email, but you can get one for your subdomain, so if you had `luke.is-a.dev` you could use luke@luke.is-a.dev. Given that you use the correct records as stated [in the docs](https://github.com/is-a-dev/register/blob/main/docs/domains-json.md#record-required).

**Reply by anime-oni** [reply]
@IbrahimDevX, here is an updated tutorial https://docs.is-a.dev/guides/zoho-mail/

**Comment by wigglycornet** [comment]
![bitmoji](https://sdk.bitmoji.com/render/panel/78048204-cb40-44e7-ace4-a0064c4af63f-dc1dd8ca-eed6-4e56-9b06-950112f7ba01-v1.png?transparent=1&palette=1&width=246) Cool!
